ApplicationLibCode/Application/RiaPreferences.h

```diff
@@ -193,6 +195,8 @@ class RiaPreferences : public caf::PdmObject
     caf::PdmField<caf::FilePath> m_lastUsedPlotTemplate;
     caf::PdmField<bool>          m_useQtChartsPlotByDefault;
 
+    caf::PdmField<QString> m_gridCalculationExpressionFolder;
+
```

Comment:
Should this be a caf::FilePath instead of QString?

Reply:
Agree, fixed